src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1469,5 +1469,8 @@ internal enum ErrorCode
         ERR_Merge_conflict_marker_encountered = 8300,
         ERR_InvalidPreprocessingSymbol = 8301,
         ERR_FeatureNotAvailableInVersion7_1 = 8302,
+
+        ERR_RuntimeDoesNotSupportDefaultInterfaceImplementation = 8401,
```

Comment:
You may want to pick a different range (8500 probably) to avoid conflict with Omar/Vlad's branch on readonly refs.